src/environment.go

```diff
@@ -170,7 +170,12 @@ func (env *environment) runCommand(command string, args ...string) (string, erro
 	}
 	out, err := exec.Command(command, args...).Output()
 	if err != nil {
```

Comment:
There's a nicer way to do this:

```golang
if exitErr, ok := err.(*exec.ExitError); ok {
    return "", &commandError{
 	err:      exitErr.Error(),
 	exitCode: exitErr.ExitCode(),
    }
}
```

Reply:
I saw this syntax but I had no idea how to use it. Thanks!